server_test.go

```diff
@@ -52,6 +52,7 @@ type ServerInterceptorTestSuite struct {
 	clientConn     *grpc.ClientConn
 	testClient     pb_testproto.TestServiceClient
 	ctx            context.Context
+	cancel         context.CancelFunc
```

Comment:
is this used anywhere? It should be atleased emptied (make sure to do `cancel()` at the tear down)